### PR TITLE
Standardize DB query normalizer property names

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**
And add normalizer configuration for Redis, Couchbase, Cassandra and Geode

Resolves: #1431

### CODE REVIEW

File: instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java [hunk 1]
```diff
@@ -36,6 +36,9 @@
   public static final io.opentelemetry.trace.Tracer TRACER =
       OpenTelemetry.getGlobalTracer("io.opentelemetry.auto.lettuce-5.1");
 
+  private static final RedisCommandNormalizer COMMAND_NORMALIZER =
+      new RedisCommandNormalizer("lettuce", "lettuce-5", "lettuce-5.1");
```
Comment (mateuszrzeszutek) [inline instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java:40]: I don't like repeating the instrumentation names here, but you can't exactly access the `Instrumenter` from the advice. Maybe it'll be possible to fix that once https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1274#issuecomment-707860035 is implemented.

Comment (trask) [reply instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java:40]: yes, that would be very nice to tie this to the existing instrumentation names 👍

**Review by trask — APPROVED** [review]
👍

File: instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java [hunk 2]
```diff
@@ -36,6 +36,9 @@
   public static final io.opentelemetry.trace.Tracer TRACER =
       OpenTelemetry.getGlobalTracer("io.opentelemetry.auto.lettuce-5.1");
 
+  private static final RedisCommandNormalizer COMMAND_NORMALIZER =
```
Comment (iNikem) [inline instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java:39]: In all other cases you have `commandNormalizer`. Please make the same here as well

**Review by iNikem — APPROVED** [review]

**Comment by iNikem** [comment]
@mateuszrzeszutek also please rebase

File: instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java (hunk 2 above)
Comment (mateuszrzeszutek) [reply instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java]: Done

**Pull request merged**